website/content/contributors/_index.md

```diff
@@ -75,11 +84,3 @@ The Cloud Native Computing Foundation projects are listed [below](projects/), to
 | ------------------------------------------------------------ | ------------------------- | ------------------ |
 |    [Cloud Native Glossary](projects/#cloud-native-glossary)        |           Definitions       |        Markdown      |     
 
-## Archived Projects
-
-{{< projects level="archived" >}}
```

Comment:
It's not clear to me why archived projects were removed? Was this a mistake?

Reply:
I've added a comment in this ticket, but it is lost.

I thought listing archived projects in a page where we encourage people to contribute to the projects is not good.
They cannot contribute to those projects anyway.

What do you think?

Reply:
Are they listed somewhere else? I am just not comfortable removing that content entirely from the website since originally the project list was owned/managed by another group of people.

Reply:
They're at least listed here: https://contribute.cncf.io/contributors/projects/#archived-projects

I didn't touch that part. Only removed the archived projects in the https://contribute.cncf.io/contributors/ page, which a person is directed to when they click "I want to be or am a Contributor" link on [the homepage](https://contribute.cncf.io/)

It is not a big deal to list them back though :) Just let me know what you think

Reply:
No thanks that helped, I'd forgotten that we had a separate page of projects listed elsewhere